lib/api/client-commands/window/minimize.js

```diff
@@ -1,4 +1,5 @@
-const ProtocolAction = require('../_base-action.js');
+const ClientCommand = require('../_base-command.js');
+const {Logger} = require('../../../utils');
```

Comment:
There are multiple places where `Logger` is unused. Can we remove those references?